### DISCUSSION TITLE
Featured extensions on the Marketplace

### DISCUSSION THREAD

**Discussion by digitarald** [microsoft/vscode-discussions#6] [Extension Show and Tell]
Please see https://github.com/microsoft/vscode/issues/141472 for what featured extensions are and comment to propose an extension that you think fits.

**Comment by macintacos** [comment]
I've been using this extension for ages, and honestly if it went away I would seriously consider moving to another app entirely (I use it that much).

That extension is VSpaceCode, and I think it deserves way more recognition than it currently receives: https://marketplace.visualstudio.com/items?itemName=VSpaceCode.vspacecode

**Comment by isidorn** [comment]
@macintacos cool idea. We can feature it the next time we update the list. Thanks
